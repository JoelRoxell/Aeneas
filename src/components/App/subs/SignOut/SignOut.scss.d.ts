// This file is generated automatically
export const center: string;
export const signOut: string;
