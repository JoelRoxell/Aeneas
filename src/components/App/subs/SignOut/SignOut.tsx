import * as React from 'react'
import { connect } from 'react-redux'
import { ConnectedReduxProps } from 'store'
import { signOut } from 'store/auth'
import { Redirect } from 'react-router'

import * as style from './SignOut.scss'

class SignOut extends React.Component<ConnectedReduxProps> {
  componentWillMount() {
    this.props.dispatch(signOut())
  }

  render() {
    return (
      <div className={style.signOut}>
        <Redirect to="/" />
      </div>
    )
  }
}

export default connect()(SignOut)
