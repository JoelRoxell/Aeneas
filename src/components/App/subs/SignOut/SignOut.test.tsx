import * as React from 'react'
import { shallow } from 'enzyme'

import SignOut from 'components/App/subs/SignOut'

describe('SignOut', () => {
  test('render', () => {
    const wrapper = shallow(<SignOut />)

    expect(wrapper).toMatchSnapshot()
  })
})
