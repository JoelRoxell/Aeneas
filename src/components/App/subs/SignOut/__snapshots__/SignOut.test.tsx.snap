// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOut render 1`] = `
<div
  className="signOut"
>
  <Redirect
    to="/"
  />
</div>
`;
